Vizzuality/marxan-cloud
# Feature Target-SPF generic component

![imagen](https://user-images.githubusercontent.com/545342/107506234-18cc9e80-6b9e-11eb-98ca-f48f3ee0b5e7.png)

### Overview

This PR adds the feature component `TargetSPF`.

### Designs

https://www.figma.com/file/hq0BZNB9fzyFSbEUgQIHdK/Marxan-Visual_V02?node-id=1523%3A2317

### Testing instructions

Go to http://192.168.1.133:6006/?path=/story/components-features-targetspf-item--default and test the different options available.

> _⚠️NOTE: For some reason the storybook interface isn't reacting when changing boolean values in a property object, e.g. `isAllTargets: true`. See the screenshot below 👇 . Thus, in order to see the other state you might need to change the `component.stories` file_
![imagen](https://user-images.githubusercontent.com/545342/107506130-f3d82b80-6b9d-11eb-9206-43c944dc6b3e.png)
 
### Feature relevant tickets
-
